CMakeLists.txt

```diff
@@ -111,38 +111,38 @@ add_subdirectory(libminifi)
 #function(createExtension extensionCondition extensionGuard extensionName description dirName)
 
 ## Add http-curl extensions
-createExtension(DISABLE_CURL 
-				HTTP-CURL 
-				"HTTP CURL" 
-				"This enables RESTProtocol, InvokeHTTP, and the HTTPClient for Site to Site" 
-				"extensions/http-curl"
-				"${TEST_DIR}/curl-tests")
+createExtension(DISABLE_CURL
+                HTTP-CURL
```

Comment:
we need to add a skip to this so this doesn't happen when the linter runs. thanks

Reply:
I don't follow. What do you need me to change?